config/routes.rb

```diff
@@ -1,7 +1,16 @@
 Myflix::Application.routes.draw do
-  root "videos#index"
+  root "pages#front"
   get '/home', to: 'videos#index'
-  resources :videos, except: [:index]
+  get '/register', to: "users#new", as: "register"
+  get '/sign_in', to: "sessions#new", as: "sign_in"
+  post '/login', to: "sessions#create"
+  get '/sign_out', to: "sessions#destroy"
```

Comment:
I would go with `delete '/sign_out'` because it is more semantically correct and no route with destroying behavior should be available for user to bookmark.